mitmproxy/command.py

```diff
@@ -0,0 +1,91 @@
+import inspect
+import typing
+import shlex
+from mitmproxy.utils import typecheck
+from mitmproxy import exceptions
+from mitmproxy import flow
+
+
+def typename(t: type, ret: bool) -> str:
+    """
+        Translates a type to an explanatory string. Ifl ret is True, we're
+        looking at a return type, else we're looking at a parameter type.
+    """
+    if t in (str, int, bool):
+        return t.__name__
+    if t == typing.Sequence[flow.Flow]:
+        return "[flow]" if ret else "flowspec"
```

Comment:
These are completely different types - it may be that we later grow internal commands that do take a list of flows as an argument. Let me think about this.